hail/python/hailtop/aiotools/fs/copier.py

```diff
@@ -163,7 +163,10 @@ def add_source_reports(transfer_report):
         print(f'  Time: {humanize_timedelta_msecs(self._duration)}')
         assert self._duration is not None
         if self._duration > 0:
-            print(f'  Average transfer rate: {humanize.naturalsize(total_bytes / (self._duration / 1000))}/s')
+            bandwidth = humanize.naturalsize(total_bytes / (self._duration / 1000))
+            print(f'  Average bandwidth: {bandwidth}/s')
+            file_rate = total_files / (self._duration / 1000)
+            print(f'  Average file rate: {file_rate:,.1f}/s')
```

Comment:
```suggestion
            print(f'  Average file rate: {file_rate:.1f}/s')
```

Reply:
Ah. Thought it was a typo. Should have dug into the docs for formatting floats more carefully.